cs/src/Connections/TunnelRelayTunnelClient.cs

```diff
@@ -127,12 +125,21 @@ protected Task ConnectAsync(string clientRelayUri, string? accessToken, Cancella
     /// </summary>
     protected virtual async Task<Stream> CreateSessionStreamAsync(CancellationToken cancellation)
     {
+        var protocols = Environment.GetEnvironmentVariable("DEVTUNNELS_PROTOCOL_VERSION") switch
```

Comment:
It is not exactly duplicated because it selects client protocols vs host protocols, and those are different constants. I don't see any reasonable way to make it shared.